src/commands/skywars.ts

```diff
@@ -23,10 +25,12 @@ export async function run(
             ),
             ephemeral: true
         });
-
-    const { data }: { data: HypixelType } = await axios(
+    // @ts-ignore
+    const { data }: { data: HypixelType } = await fetch(
```

Comment:
change this to
```ts
const data = (await fetch(
        `https://api.hypixel.net/player?uuid=${uuid.data.id}&key=${client.config.hypixel}`
    ).then(res => res.json())) as HypixelType;
```
just like you did with bedwars and remove that `@ts-ignore`

Reply:
Thats

Reply:
The ts ignore is needed cause it does type {} funni missing

Reply:
oh